mslice/plotting/plot_window/cut_plot.py

```diff
@@ -0,0 +1,314 @@
+from itertools import chain
+
+from matplotlib.container import ErrorbarContainer
+from matplotlib.legend import Legend
+from matplotlib.lines import Line2D
+import numpy as np
+
+from mslice.presenters.plot_options_presenter import CutPlotOptionsPresenter
+from mslice.presenters.quick_options_presenter import quick_options
+from .plot_options import CutPlotOptions
+
+
+class CutPlot(object):
+
+    def __init__(self, plot_figure, canvas, cut_plotter):
+        self.plot_figure = plot_figure
+        self._canvas = canvas
+        self._cut_plotter = cut_plotter
+        self._lines_visible = {}
+        self._legends_shown = True
+        self._legends_visible = []
+        self._legend_dict = {}
+        self._lines = self.line_containers()
+        plot_figure.menuIntensity.setDisabled(True)
+        plot_figure.menuInformation.setDisabled(True)
+        np.seterr(invalid='ignore')
```

Comment:
Can we avoid setting this globally?

Reply:
This was added to avoid errors in the console when changing the axis scale to logarithmic. I tried only ignoring the errors within the `change_axis_scale` method, but it still prints the errors in the next subsequent call to `canvas.draw()`.
I agree it would be better not to set this globally, but I'm not sure how to prevent the errors being printed (short of disabling them every single time draw is called)